### ISSUE TITLE
RealisticWorldGenerator compatibility

### ISSUE THREAD

**Issue by SenseiMunchkin** [PluginBugs/Issues-ItemsAdder#1386]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

SenseiMunchkin#1434

### Describe the solution you'd like

I'm currently working with RealisticWorldGenerator, and I'm generating RED_CONCRETE blocks in the caves using ores.yml. The generation works as it should, and the next part of the system is to have ItemsAdder use a world-populator to replace these RED_CONCRETE blocks with my custom block, aluminum_ore. Here is the configuration:
```
info:
  namespace: cosmicdrift
  
items:
  aluminum_ore:
    display_name: "Aluminum Ore"
    permission: cosmicdrift
    resource:
      generate: true
      textures: 
        - ores/aluminum_ore.png
      material: PAPER
    specific_properties:
      block:
        placed_model:
          type: REAL
          break_particles: BLOCK
        break_tools_whitelist:
         - STONE_PICKAXE
         - IRON_PICKAXE
    events:
      block_break: 
        drop_exp:
          chance: 100
          min_amount: 10 
          max_amount: 15
worlds_populators:
  aluminum_ore:
    block: cosmicdrift:aluminum_ore
    worlds:
      - aluminum
    replaceable_blocks:
      - RED_CONCRETE
    chance: 100.0
    max_height: 67
    min_height: 0
    amount: 6
    iterations: 1
```

This works to an extent, I enabled the debug logging and I do see RED_CONCRETE being replaced, however it is only replacing some of the RED_CONCRETE present in a chunk. My intention is to have each and every RED_CONCRETE block be turned into Aluminum Ore. That's my feature request, the ability to have an option to override the chance option and replace all blocks. 

Things I've Tried:
- Generated the texture with RWG (using mushroom attributes), the textures loaded but the block wouldn't drop the ore, since it wasn't registered with ItemsAdder
- Using WorldBorder chunk generation (I received debug logs as normal, but still not every RED_CONCRETE).
- Manual chunk generation (Same story as WorldBorder)
- Setting 'amount' and 'iteration' to insane amounts (Didn't change much at all)
- Setting 'chance' to insane amounts (Didn't change much at all either)
- Having RWG load 'itemsadder:cosmicdrift:aluminum_ore' and 'cosmicdrift:aluminum_ore" (I got an error saying there was no block placer)
- Using Dirt/Stone at a 100 chance (Same result, or just utterly broken. Aluminum Ore was in the sky).

Basically I spent hours, and I firmly believe that if it was possible I would've found it.

### Is your feature request related to a problem?

I don't believe it's a problem with the code.

### Describe alternatives you've considered

An alternative I've considered is a feature that replaces the generated mushroom blocks (from RWG) with the custom attribute combinations with ItemsAdder registered blocks, however according to another Feature Request this is coming in a future update, but it isn't here now and I believe that a 'replace-all' feature would still be good

### Additional context

Not that I can think of, but I'm always open to talk on discord about what I've tried and the details of the project I couldn't think of without being asked.

**Comment by LoneDev6** [comment]
Another solution: restore RWG native support, I'll work on this task this week
